### DISCUSSION TITLE
I want to start my modified source code through docker-compose.yml, but looking at the startup output, it is still running through dockerhub pull image. How can I achieve what I want?

### DISCUSSION THREAD

**Discussion by kuschzzp** [langgenius/dify#8042] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

I want to start my modified source code through docker-compose.yml, but looking at the startup output, it is still running through dockerhub pull image. How can I achieve what I want?

**Comment by laipz8200** [comment]
If you make changes to the code, you should start from the source code. Alternatively, you can build your own image from the modified source code and update the Docker Compose configuration file.
